api/task_check_run.go

```diff
@@ -98,12 +99,23 @@ type TaskCheckDatabaseStatementAdvisePayload struct {
 	PolicyID int `json:"policyID,omitempty"`
 }
 
+// Namespace is the namespace for task check result.
+type Namespace string
+
+const (
+	// AdvisorNamespace is task check result namespace for advisor.
+	AdvisorNamespace Namespace = "advisor"
+	// BBNamespace is task check result namespace for bytebase.
+	BBNamespace Namespace = "bb"
```

Comment:
I am thinking to use "bb.core" and "bb.advisor" respectively since they both maintained by us.

It could be possible another company develops "foo.advisor" later